lib/simplecov-html.rb

```diff
@@ -70,9 +74,26 @@ def asset_output_path
       end
 
       def assets_path(name)
+        return asset_inline(name) if @inline_assets
+
         File.join("./assets", SimpleCov::Formatter::HTMLFormatter::VERSION, name)
       end
 
+      def asset_inline(name)
+        path = File.join(@public_assets_dir, name)
+
+        # Only have a few content types, just hardcode them
+        content_type = {
+          ".js" => "text/javascript",
+          ".png" => "image/png",
+          ".gif" => "image/gif",
+          ".css" => "text/css",
+        }[File.extname(name)]
+
+        base64_content = Base64.strict_encode64 File.open(path).read
```

Comment:
Can you put a `require "base64"` on top of the file? I got a `NameError` when I tried to run.

Reply:
hey, sorry i missed this comment 

better 3 years late than never I guess?